### PR TITLE
Updated styles for the editor

### PR DESCRIPTION
**Pull request by Matthew-Burfield**
Still haven't found a great solution for theming yet, but I think we're going to have to use SASS or LESS or STYLUS

I played around with stylus today and was able to set it up with the webpack loaders, but I don't know how to make global variables. I think I'll create a new issue for this.

Closes issue #27
Closes issue #28

### CODE REVIEW

File: client/src/components/Home/components/Editor.vue [hunk 1]
```diff
@@ -47,12 +47,17 @@ h2 {
...
-  width: 500px;
-  height: 500px;
+  background-color: #1e2127;
+  color: #9ba1a2;
+  width: 100%;
+  height: 100%;
   white-space: pre;
+  font-size: 16px;
+  text-align: left;
+  padding-left: 60px;
```
Comment (AlysCole) [inline client/src/components/Home/components/Editor.vue:57]: We might want to use a `float: right` for the editor once the sidebar (with a `float: left`) is added.

`flex` or `table` row styles would work, as well. Maybe even more so, to make sure the appropriate line number is aligned with the line it refers to.

File: client/src/styles/colors.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  background: '#272727',
+  icon: '#878787',
+  iconActive: '#FFFFFF',
+  iconDisabled: '#878787',
+};
+
+export default {
+  editor,
+  sidebar,
+};
```
Comment (AlysCole) [inline client/src/styles/colors.js:17]: Are these some of the global variables you mentioned?

If we're using Stylus, we can make use of [`.define`](http://stylus-lang.com/docs/js.html#definename-node) for global variables.

**Review by AlysCole — APPROVED** [review]

File: client/src/components/Home/components/Editor.vue (hunk 1 above)
Comment (Matthew-Burfield) [reply client/src/components/Home/components/Editor.vue:57]: My plan is to absolute position the sidebar to the left, and then the editor can take up the rest of the screen with padding over the sidebar

File: client/src/styles/colors.js (hunk 2 above)
Comment (Matthew-Burfield) [reply client/src/styles/colors.js:17]: Oh do you know how to use stylus? I've never used it before so had no idea how to define global variables

**Pull request merged**